docs/DeveloperGuide.md

```diff
@@ -1237,38 +1237,59 @@ testers are expected to do more *exploratory* testing.
 
    1. Download the jar file and copy into an empty folder
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   2. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
-1. Saving window preferences
+2. Saving window preferences
 
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
-   1. Re-launch the app by double-clicking the jar file.<br>
+   2. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
-
-### Deleting a person
-
-1. Deleting a person while all persons are being shown
-
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
-
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
-
-   1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
-
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
-
-1. _{ more test cases …​ }_
-
 ### Saving data
 
 1. Dealing with missing/corrupted data files
 
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
-
-1. _{ more test cases …​ }_
+   1. Test case: Missing data file<br>
+      Expected: Default contacts are loaded instead. Upon an operation that interacts with the data, e.g. add/delete/edit persons, the data will then be saved as `addressbook.json`.
+   
+   2. Test case: Corrupted JSON Formatting in data file<br>
+      Expected: An empty AddressBook will be loaded instead. Upon an operation that interacts with the data, the existing corrupted `addressbook.json` file will be replaced with the new data.
+   
+   3. Test case: Invalid entry in individual Person data in data file (e.g. alphabets in phone number)<br>
+      Expected: Data is loaded into ContaX normally, with invalid entries skipped.
+
+2. Manually editing JSON entries
+
+   1. Prerequisites: Do not have an active instance of ContaX open.
+   
+   2. Test case: Editing the entries directly from the JSON file <br>
+      Expected: Upon opening of ContaX, the previously edited data is loaded into ContaX normally.
+   
+   3. Test case: Editing the entries but with invalid data <br>
+      Expected: Upon opening of ContaX, the particular entry will be skipped.
+
+### Importing Contacts from CSV Files
+
+1. Importing contacts from a CSV File
+
+   1. Prerequisites: Must have an existing CSV File with contact information.
+   
+   2. Test case: CSV File was exported previously from another instance of ContaX, or is in proper ContaX format <br>
```

Comment:
I suggest linking to the CSV format section.

Reply:
done in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/341/commits/c96996c17298418ba564eb619e4b83c4cb7e62c3